dotnet/src/extensions/Azure/Handlers/HttpHandler/HttpTriggerHandler.cs

```diff
@@ -47,6 +52,63 @@ public HttpTriggerHandler(IGXRouting gxRouting)
 			return httpResponseData;
 		}
 	}
+
+	public class RedisHttpSession : ISession
+	{
+		const int AZURE_SESSION_TIMEOUT_IN_MINUTES = 5;
```

Comment:
This 5 minutes settings means that all Redis Data will expire after 5 minutes?
If yes, then how do I specify Redis Session Timeout expiration?